bug_update.php

```diff
@@ -307,6 +307,11 @@
 	$t_new_custom_field_value = gpc_get_custom_field( 'custom_field_' . $t_cf_id, $t_cf_def['type'], '' );
 	$t_old_custom_field_value = custom_field_get_value( $t_cf_id, $f_bug_id );
 
+	# No need to update the field if its value has not changed
+	if( $t_old_custom_field_value == $t_new_custom_field_value ) {
```

Comment:
What happens when old value is say 1 and new value is '1'?

Reply:
Then add a parameter to history_log_event_direct() that forces the strict comparison?